aten/src/ATen/native/RNN.cpp

```diff
@@ -1168,7 +1280,12 @@ std::tuple<Tensor, Tensor, Tensor> quantized_lstm(
       "dtype is not supported");
 
   std::tuple<Tensor, Tensor, Tensor> results;
-  if (result_dtype == at::kChar || result_dtype == at::kQInt8) {
+  if (result_dtype == at::kQInt8 && !use_dynamic) {
```

Comment:
@jamesr66a  This is a check for the future -- `if qint8 and not use_dynamic: use static`